SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -6,19 +6,20 @@ namespace SeleniumPracticalExercise.PageObjects
 {
     public class EmployeeInformationPage : BasePageLocal
     {
-        
+        private readonly By _actualFirstNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");
+        private readonly By _actualIDLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][2]/div");
+        private readonly By _actualLastNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][4]/div");
         private readonly By _addButtonLocator = By.CssSelector("div.orangehrm-header-container > button");
-        private readonly By _employeeIDTxtBoxLocator = By.CssSelector("div.oxd-input-group input.oxd-input");  //EmployeeInfoPage
-        private readonly By _expectedIDLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][2]/div");
-        private readonly By _expectedFirstNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");
-        private readonly By _expectedLastNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][4]/div");
+        private readonly By _employeeIDTxtBoxLocator = By.CssSelector("div.oxd-input-group input.oxd-input"); 
         private readonly By _searchButtonLocator = By.CssSelector("button[type='submit']");
 
         public EmployeeInformationPage(IWebDriver driver) : base(driver)
         {
         }
 
         /// <summary>
+        /// PIM button
```

Comment:
This XML doc for a method should only describe what the method does. For example, in this case "Clicks the Add button" or whatever. The rest should be removed here and elsewhere.

Reply:
update my XML and was moved to the LeftNavPanel page object model